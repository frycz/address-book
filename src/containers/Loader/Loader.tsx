import React from "react";

import "./Loader.scss";

/**
 * Loader displayed when lazy modules are fetched
 */
export const Loader: React.FC = () => (
  <div className="loader-box">
    Loading...
  </div>
);

export default Loader;
