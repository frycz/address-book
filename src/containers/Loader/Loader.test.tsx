import React from "react";
import { shallow } from "enzyme";
import { Loader } from "./Loader";

describe("Loader", () => {
  it("should match smapshot", () => {
    const loader = shallow(<Loader />);
    expect(loader.debug()).toMatchSnapshot();
  });
});
