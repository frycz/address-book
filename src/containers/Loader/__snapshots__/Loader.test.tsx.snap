// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match smapshot 1`] = `
"<div className=\\"loader-box\\">
  Loading...
</div>"
`;
